app/scripts/modules/kubernetes/securityGroup/configure/wizard/upsert.controller.js

```diff
@@ -181,6 +183,14 @@ module.exports = angular.module('spinnaker.securityGroup.kubernetes.create.contr
             cloudProvider: 'kubernetes',
             region: $scope.securityGroup.namespace,
           };
+
+          // Change TLS hosts from string to array for Clouddriver (if it isn't already an array)
+          for (let idx in $scope.securityGroup.tls) {
+            if (!Array.isArray($scope.securityGroup.tls[idx].hosts.constructor)) {
```

Comment:
I should have been more clear. I think this is always false -- `Array.isArray([]) === true`, but `Array.isArray([].constructor]) === false`.

Should be `!Array.isArray($scope.securityGroup.tls[idx].hosts)`.

Reply:
ah sorry, should've caught that